### PR TITLE
Create React Redux Assignment

### PR DESCRIPTION
**Pull request by madan-webonise**
- Assignment(HelloWorld, TodoList)
- Assignmnet (webpack configuration)

### CODE REVIEW

File: my-app/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "react-dom": "^16.3.1",
+    "react-scripts": "1.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  }
+}
```
Comment (swapnil-webonise) [inline my-app/package.json]: Configure your editor for adding a new line.

File: my-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import './App.css';
+import './assets/styles/layout.css';
```
Comment (swapnil-webonise) [inline my-app/src/App.js]: Use alias for importing dependencies

File: my-app/src/App.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import TodoList from './TodoList';
+
+class App extends Component {
```
Comment (swapnil-webonise) [inline my-app/src/App.js]: give component name like AppComponent

**Review by swapnil-webonise — CHANGES_REQUESTED** [review]

File: my-app/src/App.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    if (insertValue !== '') {
+      tempArray.push(insertValue);
+      this.setState({
+        todoArray: tempArray
+      });
+    }
+    insertElement.value = '';
+  }
+
+//delete functionality has to be done
```
Comment (owais-webonise) [inline my-app/src/App.js]: Remove comments

File: my-app/src/TodoList.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './assets/styles/layout.css';
+
+class TodoList extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      arrayFromParent: this.props.displayList
```
Comment (owais-webonise) [inline my-app/src/TodoList.js]: Change name of array, use proper name convention

File: my-app/src/TodoList.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+                </li>
+            );
+          })
+        }
+      </ul>
+    );
+  }
+}
+
+export default TodoList;
```
Comment (owais-webonise) [inline my-app/src/TodoList.js]: Add default props

**Review by owais-webonise — CHANGES_REQUESTED** [review]
Needs Improvement

**Review by owais-webonise — APPROVED** [review]

**Pull request merged**